pretrained_bert/dataset_reader.py

```diff
@@ -140,6 +140,7 @@ def is_whitespace(char):
                 break
             start_offset += min(length, self._document_stride)
 
+        #
         for (doc_span_index, doc_span) in enumerate(doc_spans):
```

Comment:
Would you mind removing this for loop altogether and just take `doc_spans[0]` to make it even clearer what's happening here?

Reply:
I have your idea and I will work on it.

Reply:
Need I create a new pull request to add this part ? (removing this for loop altogether...)

Reply:
I have removed unused loop logic to make the code clear. Feel free to contact me if you have any suggestions.